.gitignore

```diff
@@ -19,7 +19,6 @@ cache/*
 embedded_gen.go
 dist
 music
-docker-compose.yml
```

Comment:
This should be changed to `contrib/docker-compose.*` instead of removed. Me and probably other devs usually have a local env docker-compose in the project's root, and removing this line makes git want to commit it.

Reply:
I almost figured. But then it has to be inverted. Just pushed the change.